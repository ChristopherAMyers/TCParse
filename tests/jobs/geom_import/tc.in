run energy
method camb3lyp
basis 6-31G*
coordinates geom.xyz
charge 0
