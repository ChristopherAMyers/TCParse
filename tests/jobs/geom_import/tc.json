{
    "atoms": [
        "O",
        "O",
        "C",
        "C",
        "C",
        "H",
        "H",
        "H",
        "H"
    ],
    "geom": [
        [
            1.369627119,
            0.951356327,
            9.908438528
        ],
        [
            -1.199303734,
            0.961400887,
            9.911224786
        ],
        [
            0.068001267,
            -1.022175749,
            10.03218435
        ],
        [
            1.253749238,
            -0.356789413,
            9.990042489
        ],
        [
            -1.156324326,
            -0.272208839,
            9.987482719
        ],
        [
            0.435660215,
            1.312868501,
            9.885783618
        ],
        [
            0.048825711,
            -2.101912493,
            10.10068293
        ],
        [
            2.212979712,
            -0.868893011,
            10.02225262
        ],
        [
            -2.104215212,
            -0.835646171,
            10.02390796
        ]
    ],
    "energy": [
        -267.0296478017
    ],
    "dipole_moment": 2.587745426388384,
    "dipole_vector": [
        0.525395,
        -2.529013,
        0.15646
    ]
}