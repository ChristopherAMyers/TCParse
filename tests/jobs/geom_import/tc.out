Startfile from command line: tc.in


       ***********************************************************
       *                    TeraChem v1.9-2022.03-dev            *
       *  Git Version: 4daa16dd21e78d64be5415f7663c3d7c2785203c  *
       *                   Development Version                   *
       *           Chemistry at the Speed of Graphics!           *
       ***********************************************************
       * This program may only be used in connection with        *
       * a valid license from PetaChem, LLC. Use of this program *
       * or results thereof indicates acceptance of all terms    *
       * and conditions stated in the license and that a valid   *
       * license agreement between the user and PetaChem, LLC    *
       * exists. PetaChem, LLC does not warrant the correctness  *
       * of results or their suitability for any purpose.        *
       * Please email bugs, suggestions, and comments to         *
       *                  help@petachem.com                      *
       *                                                         *
       ***********************************************************


       ***********************************************************
       *  Compiled by coltonbh     Sat Mar 26 12:11:31 PDT 2022  *
       *  Supported architectures: sm_37 sm_52 sm_60 sm_61 sm_70 sm_75 sm_80 sm_86 sm_87*
       *  Build cuda_11.4.r11.4/compiler.30188945_0              *
       ***********************************************************


 Job started   Fri Jul 19 16:13:58 2024
 On node004 (available memory: 64642 MB)

######################################### RUNTIME INFO ##########################################
/terachem/bin/terachem tc.in 

NVRM version: NVIDIA UNIX x86_64 Kernel Module  470.57.02  Tue Jul 13 16:14:05 UTC 2021
GCC version:  gcc version 4.8.5 20150623 (Red Hat 4.8.5-44) (GCC) 

	linux-vdso.so.1 =>  (0x00002aaaaaacd000)
	libcurl.so.4 => /usr/lib64/libcurl.so.4 (0x00002aaaaaccf000)
	libmpi.so.12 => /terachem/external_libs/libmpi.so.12 (0x00002aaaaaf39000)
	libdftbplus.so => /terachem/lib/libdftbplus.so (0x00002aaaabc6f000)
	libfsockets.so => /terachem/lib/libfsockets.so (0x00002aaaac155000)
	libxmlf90.so => /terachem/lib/libxmlf90.so (0x00002aaaac359000)
	libthcbox.so.1 => /terachem/lib/libthcbox.so.1 (0x00002aaaac57c000)
	libOpenMM.so => /terachem/external_libs/libOpenMM.so (0x00002aaaaf913000)
	libgpubox.so.1 => /terachem/lib/libgpubox.so.1 (0x00002aaaaff77000)
	libtensorbox.so.1 => /terachem/lib/libtensorbox.so.1 (0x00002aaab018a000)
	libiobox.so.1 => /terachem/lib/libiobox.so.1 (0x00002aaab03a6000)
	libintbox.so.1 => /terachem/lib/libintbox.so.1 (0x00002aaab05b6000)
	libsqmbox.so.1 => /terachem/lib/libsqmbox.so.1 (0x00002aaaf1e1f000)
	libgridbox.so.1 => /terachem/lib/libgridbox.so.1 (0x00002aaaf2386000)
	libdftbox.so.1 => /terachem/lib/libdftbox.so.1 (0x00002aaaf2b80000)
	libcibox.so.1 => /terachem/lib/libcibox.so.1 (0x00002aaaf2e65000)
	libccbox.so.1 => /terachem/lib/libccbox.so.1 (0x00002aaaf3a21000)
	libsimbox.so.1 => /terachem/lib/libsimbox.so.1 (0x00002aaaf40ee000)
	liboptbox.so.1 => /terachem/lib/liboptbox.so.1 (0x00002aaaf430b000)
	libprotobuf.so.25 => /terachem/external_libs/libprotobuf.so.25 (0x00002aaaf453f000)
	libcommbox.so.1 => /terachem/lib/libcommbox.so.1 (0x00002aaaf49f2000)
	libcuda.so.1 => /.singularity.d/libs/libcuda.so.1 (0x00002aaaf4c57000)
	libcudart.so.11.0 => /terachem/lib/libcudart.so.11.0 (0x00002aaaf63b3000)
	libcublas.so.11 => /terachem/lib/libcublas.so.11 (0x00002aaaf6655000)
	libcufft.so.10 => /terachem/lib/libcufft.so.10 (0x00002aaaff58e000)
	libcusparse.so.11 => /terachem/lib/libcusparse.so.11 (0x00002aab1520e000)
	libcusolver.so.11 => /terachem/lib/libcusolver.so.11 (0x00002aab235f9000)
	libcrypto.so.10 => /usr/lib64/libcrypto.so.10 (0x00002aab308c2000)
	libssl.so.10 => /usr/lib64/libssl.so.10 (0x00002aab30d25000)
	libstdc++.so.6 => /usr/lib64/libstdc++.so.6 (0x00002aab30f97000)
	libm.so.6 => /usr/lib64/libm.so.6 (0x00002aab3129f000)
	libcilkrts.so.5 => /terachem/lib/libcilkrts.so.5 (0x00002aab315a1000)
	libgcc_s.so.1 => /usr/lib64/libgcc_s.so.1 (0x00002aab317dc000)
	libpthread.so.0 => /usr/lib64/libpthread.so.0 (0x00002aab319f2000)
	libc.so.6 => /usr/lib64/libc.so.6 (0x00002aab31c0e000)
	/lib64/ld-linux-x86-64.so.2 (0x00002aaaaaaab000)
	libdl.so.2 => /usr/lib64/libdl.so.2 (0x00002aab31fdc000)
	libidn.so.11 => /usr/lib64/libidn.so.11 (0x00002aab321e0000)
	libssh2.so.1 => /usr/lib64/libssh2.so.1 (0x00002aab32413000)
	libssl3.so => /usr/lib64/libssl3.so (0x00002aab32640000)
	libsmime3.so => /usr/lib64/libsmime3.so (0x00002aab328a3000)
	libnss3.so => /usr/lib64/libnss3.so (0x00002aab32acb000)
	libnssutil3.so => /usr/lib64/libnssutil3.so (0x00002aab32e04000)
	libplds4.so => /usr/lib64/libplds4.so (0x00002aab33034000)
	libplc4.so => /usr/lib64/libplc4.so (0x00002aab33238000)
	libnspr4.so => /usr/lib64/libnspr4.so (0x00002aab3343d000)
	libgssapi_krb5.so.2 => /usr/lib64/libgssapi_krb5.so.2 (0x00002aab3367b000)
	libkrb5.so.3 => /usr/lib64/libkrb5.so.3 (0x00002aab338c8000)
	libk5crypto.so.3 => /usr/lib64/libk5crypto.so.3 (0x00002aab33bb1000)
	libcom_err.so.2 => /usr/lib64/libcom_err.so.2 (0x00002aab33de4000)
	liblber-2.4.so.2 => /usr/lib64/liblber-2.4.so.2 (0x00002aab33fe8000)
	libldap-2.4.so.2 => /usr/lib64/libldap-2.4.so.2 (0x00002aab341f7000)
	libz.so.1 => /usr/lib64/libz.so.1 (0x00002aab3444c000)
	librt.so.1 => /usr/lib64/librt.so.1 (0x00002aab34662000)
	libifport.so.5 => /terachem/external_libs/libifport.so.5 (0x00002aab3486a000)
	libifcoremt.so.5 => /terachem/external_libs/libifcoremt.so.5 (0x00002aab34a99000)
	libimf.so => /terachem/external_libs/libimf.so (0x00002aab34e29000)
	libsvml.so => /terachem/external_libs/libsvml.so (0x00002aab35316000)
	libiomp5.so => /terachem/external_libs/libiomp5.so (0x00002aab36234000)
	libintlc.so.5 => /terachem/external_libs/libintlc.so.5 (0x00002aab365d8000)
	libirng.so => /terachem/external_libs/libirng.so (0x00002aab36844000)
	libcublasLt.so.11 => /usr/local/cuda-11.4/lib64/libcublasLt.so.11 (0x00002aab36bb7000)
	libkrb5support.so.0 => /usr/lib64/libkrb5support.so.0 (0x00002aab4a5b6000)
	libkeyutils.so.1 => /usr/lib64/libkeyutils.so.1 (0x00002aab4a7c6000)
	libresolv.so.2 => /usr/lib64/libresolv.so.2 (0x00002aab4a9ca000)
	libsasl2.so.3 => /usr/lib64/libsasl2.so.3 (0x00002aab4abe4000)
	libselinux.so.1 => /usr/lib64/libselinux.so.1 (0x00002aab4ae01000)
	libcrypt.so.1 => /usr/lib64/libcrypt.so.1 (0x00002aab4b028000)
	libpcre.so.1 => /usr/lib64/libpcre.so.1 (0x00002aab4b25f000)
	libfreebl3.so => /usr/lib64/libfreebl3.so (0x00002aab4b4c1000)
#################################################################################################

Cannot find license.dat file in the TeraChem installation directory /terachem


TRYING THE NETWORK LICENSE...
Connecting to license server '54.208.252.40' port '8877'...
Jobname: geom
Scratch directory: ./scr.geom
Random number seed: 1371805834

XYZ coordinates geom.xyz
Molden File Output: ./scr.geom/geom.molden
Using basis set: 6-31gs
dmrgstart not found
Spin multiplicity: 1
DIIS will use up to 10 vectors.
Condition number limit for the DIIS overlap matrix is            0
WF convergence threshold: 3.00e-05
Using DIIS algorithm to converge WF
Maximum number of SCF iterations: 100
Incremental fock with rebuild every 8 iterations
Will switch to conventional Fock if diffuse functions are detected
X-matrix tolerance: 1.00e-04
PRECISION: DYNAMIC
TeraChem will select linear algebra engine
DFT Functional requested: camb3lyp
Method: CAM-B3LYP 
  Slater exchange functional:           0.35 
  Full range Becke 1988 exchange:       0.35 
  Full range Hartree-Fock exchange:     0.19 
  SR Becke 1988 exchange functional:    0.46 
  LR Hartree-Fock exchange:             0.46 
  Range Correction Scaling:             0.330 a.u. 
  Lee-Yang-Parr correlation functional: 0.81 
  VWN(V) correlation functional:        0.19 
Wavefunction: RESTRICTED
DFT grid type: 1
Using dynamic DFT grids.
Initial guess generated by maximum overlap

********************************************
***** SINGLE POINT ENERGY CALCULATIONS *****
********************************************
Starting TensorBox...
TeraChem will select linear algebra engine
Using cuSolver for matrix diagonalization

using 4 out of 4 CUDA devices
   Device 0:            Tesla K80, 11441MB, CC 3.7 -- CPU THREAD 0
   Device 1:            Tesla K80, 11441MB, CC 3.7 -- CPU THREAD 1
   Device 2:            Tesla K80, 11441MB, CC 3.7 -- CPU THREAD 2
   Device 3:            Tesla K80, 11441MB, CC 3.7 -- CPU THREAD 3
-------------------------------------------------------------------
 CPU Memory Available: 32230.75 MegaWords 
 GPU Memory Available: 1174.16 MegaWords 
 Maximum recommended basis set size: 12200 basis functions
 (limited by GPU memory)
-------------------------------------------------------------------
Initialize GPUBox with 128 MW of memory
Connected!
Checking your license...

**************************************************************
  Greetings, Martinez Group! You have 1000 licenses in total
  IN USE: 32
  AVAILABLE: 968
**************************************************************


****** QM coordinates ******
O         1.3696271192         0.9513563270        -0.0915614720  
O        -1.1993037341         0.9614008865        -0.0887752139  
C         0.0680012668        -1.0221757492         0.0321843472  
C         1.2537492375        -0.3567894133        -0.0099575109  
C        -1.1563243262        -0.2722088385        -0.0125172808  
H         0.4356602153         1.3128685008        -0.1142163820  
H         0.0488257112        -2.1019124933         0.1006829284  
H         2.2129797120        -0.8688930107         0.0222526243  
H        -2.1042152118        -0.8356461709         0.0239079573  

Basis set:       6-31gs
Total atoms:     9
Total charge:    0
Total electrons: 38 (19-alpha, 19-beta)
Number electrons modeled by ECPs: 0
Total orbitals:  83
Total AO shells: 38 (23 S-shells; 10 P-shells; 5 D-shells; 0 F-shells; 0 G-shells)
Spin multiplicity: 1
Nuclear repulsion energy (QM atoms): 165.525607708879 a.u.

Setting up the DFT grid...
time to set the grid = 0.01 s
DFT grid points: 9111 (1012 points/atom)
Setting up the DFT grid...
time to set the grid = 0.01 s
DFT grid points: 25595 (2843 points/atom)

                      *** Start SCF Iterations ***

 Iter     DIIS Error       Energy change      Electrons        XC Energy           Energy        Time(s)
----------------------------------------------------------------------------------------------------------
Purifying guess...Density matrix is not well-conditioned, rescale and purify
Purification run on CPU (double precision)  using TC2 purification.
Density matrix purification finished after a total of 14 matrix multiplies
THRESPDP set to 1.00e+00
                >>> SWITCHING TO GRID 1 <<<
   1     0.3602921892    -265.3953112826    37.9857817293   -24.3836935890    -265.3953112826     0.15  
                >>> SWITCHING TO GRID 0 <<<
   2     0.2138415208      -1.1617810139    37.9994088186   -26.1261774367    -266.5570922965     0.53  
   3     0.0708176886      -0.4296727022    37.9991599592   -25.4156112791    -266.9867649988     0.10  
   4     0.0511782742      -0.0276236864    37.9990888011   -25.4097864669    -267.0143886852     0.52  
   5     0.0052987872      -0.0153372400    37.9991440358   -25.4650544869    -267.0297259252     0.47  
                >>> SWITCHING TO GRID 1 <<<
   6     0.0051905415      +0.0002279936    38.0000096924   -25.4532094425    -267.0294979316     0.11  
THRESPDP set to 1.42e-02
   7     0.0004874488      -0.0001453728    38.0000097097   -25.4589493558    -267.0296433044     0.12  
   8     0.0001455886      -0.0000039000    38.0000096918   -25.4588069066    -267.0296472044     0.12  
   9     0.0000704074      -0.0000004229    38.0000097298   -25.4587730980    -267.0296476273     0.14  
  10     0.0000267439      -0.0000001744    38.0000096606   -25.4587301839    -267.0296478017     0.14  
----------------------------------------------------------------------------------------------------------
FINAL ENERGY: -267.0296478017 a.u.
WARNING: Final energy is higher than the lowest energy by   7.8124e-05.
HOMO-LUMO GAP: 8.225 eV
CENTER OF MASS: {0.073691, 0.114789, -0.037976} ANGS
DIPOLE MOMENT: {0.525395, -2.529013, 0.156460} (|D| = 2.587746) DEBYE
Writing out molden info

Running Mulliken population analysis...

Total processing time: 2.45 sec 

 Job finished: Fri Jul 19 16:14:12 2024

